.github/workflows/e2e.yml

```diff
@@ -95,6 +95,18 @@ jobs:
 
             - run: pnpm --filter core exec playwright install chromium --with-deps
 
+            - name: Run playwright on context-editor
+              id: editorTests
+              run: pnpm --filter context-editor run playwright:test
+
+            - name: Upload playwright snapshots artifact
+              if: failure() && steps.editorTests.outcome == 'failure'
+              uses: actions/upload-artifact@v4
+              with:
+                  name: playwright-snapshots-editor
+                  path: packages/context-editor/test-results
+                  retention-days: 10
+
```

Comment:
I put this here since the test had already set up all the dependencies (including playwright) but let me know if we should split it out to a different workflow instead?

Reply:
i think this is fine! the context editor tests are likely to be much quicker than the regular e2e tests as they won't have to wait for the server to respond, so running it first is a good call. 
i think we could even move it up a bit more, before we setup the db and such.

ideally, we'd run the context-editor tests in parallel with the other tests, but afaik its not trivial to run steps in parallel